test/spec/unit/core/targeting_spec.js

```diff
@@ -416,6 +416,46 @@ describe('targeting tests', function () {
       });
     });
 
+    describe('targetingControls.allowTargetingKeys', function () {
```

Comment:
If we are going to use "begins with" matching then embody that in a test as a form of explicit documentation, this will trigger a test failure if someone tries to "improve" the code and change the behavior among other advantages like documenting the reality in the test:

it('should allow hn_pb and hb_pb_anything in any variation')

Reply:
Makes sense. The behavior is being tested, but it's not documented clearly. Thanks!

Reply:
This has been updated.